### ISSUE TITLE
when the copilot chat returns a list of ffiles as reference, the list is not accessible to NVDA or JAWS

### ISSUE THREAD

**Issue by rperez030** [microsoft/vscode-copilot-release#756]
Type: <b>Bug</b>

This is likely reproducible with any screen reader, but I haven't had time to test with all.

Steps to reproduce:

1. In copilot chat, send a query using the workspace command so that a list of files is returned as part of the answer. It is best if the query returns more than one file.
2. With NVDA or JAWS running, press CTRL+Up arrow to move to the list of messages.
3. With focus in the list of messages, use the Tab key to move focus to the referenced files and press Enter to expand.
4. Use the Tab key to focus on the list of files and move through the list with the arrow keys.

Current behavior: JAWS reads the position information for each file but not the information about the file itself.

NVDA reads the entire list of files when the list receives focus, but while navigating through the list, it reads just "list item" for each file and not the file information.

Expected behavior:
The information for each file should be announced when the file receives focus.

My guess in this case is that the keyboard focus is being placed in the <li> tag.

Extension version: 0.12.2024011602
VS Code version: Code - Insiders 1.86.0-insider (2c8f29174a63b3ace3d00ad0028cdc7ed9853b8b, 2024-01-16T05:37:21.421Z)
OS version: Windows_NT arm64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple Silicon (6 x 3200)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: unavailable_off<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.99GB (8.97GB free)|
|Process Argv|--crash-reporter-id 286be978-64ae-4658-8bf9-a3235a9d2cd2|
|Screen Reader|yes|
|VM|67%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
vsaa593:30376534
py29gd2263:30784851
vsclangdf:30492506
c4g48928:30535728
pythongtdpath:30726887
i26e3531:30792625
welcomedialogc:30812479
pythonidxpt:30768918
pythonnoceb:30776497
asynctok:30898717
dsvsc013:30777762
dsvsc014:30777825
dsvsc015:30821418
pythontestfixt:30866404
pythonregdiag2:30926734
pyreplss1:30879911
pythonmypyd1:30859725
pythoncet0:30859736
pythontbext0:30879054
accentitlementst:30870582
dsvsc016:30879898
dsvsc017:30880771
dsvsc018:30880772

```

</details>

**Comment by rperez030** [comment]
@meganrogge
